### PR TITLE
Add Meteringpoint Type To Roles

### PR DESCRIPTION
**Pull request by FirestarJes**
Add meteringpoint types to Roles, adds integration tests etc.

### CODE REVIEW

File: source/Energinet.DataHub.MarketParticipant.Domain/Model/IOrganizationRole.cs [hunk 1]
```diff
@@ -47,24 +47,23 @@ public interface IOrganizationRole
...
 
+        /// <summary>
+        ///     The status of the current role.
+        /// </summary>
+        public ICollection<MeteringPointType> MeteringPointTypes { get; }
+
         /// <summary>
         /// Activates the current role, the status changes to Active.
-        /// Only New, Inactive and Passive roles can be activated.
+        /// Only New and Inactive roles can be activated.
```
Comment (defectiveAi) [inline source/Energinet.DataHub.MarketParticipant.Domain/Model/IOrganizationRole.cs:57]: Merge error, we need Passive state.

File: source/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/GridAreaRepository.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@
 using Energinet.DataHub.MarketParticipant.Infrastructure.Persistence.Mappers;
 using Energinet.DataHub.MarketParticipant.Infrastructure.Persistence.Model;
 using Energinet.DataHub.MarketParticipant.Utilities;
+using Microsoft.EntityFrameworkCore;
```
Comment (defectiveAi) [inline source/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/GridAreaRepository.cs:21]: The shame (unused using)

Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/Repositories/GridAreaRepository.cs:21]: Such using, much shame. Fixed

File: source/Energinet.DataHub.MarketParticipant.Domain/Model/IOrganizationRole.cs (hunk 1 above)
Comment (defectiveAi) [reply source/Energinet.DataHub.MarketParticipant.Domain/Model/IOrganizationRole.cs:57]: You need to push? Should these changes not be visible anymore?

Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.Domain/Model/IOrganizationRole.cs]: I had forgotten to update the documentation for these functions, it is fixed now

**Review by defectiveAi — DISMISSED** [review]

**Review by michaelmajgaard — DISMISSED** [review]

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**